gis/qlandkartegt/Portfile

```diff
@@ -2,15 +2,16 @@
 
 PortSystem       1.0
 PortGroup        cmake 1.0
+PortGroup        github 1.0
 PortGroup        qt4 1.0
 
 name             qlandkartegt
-version          1.8.1
-revision         6
+version          1.8.1-macports-20190301
```

Comment:
There is no need to duplicate the `version` field. Just directly put the version into the `github.setup` line.